### ISSUE TITLE
[Bug]: deprecated punycode error, while running shopify commands

### ISSUE THREAD

**Issue by ivanafanasyeu** [Shopify/cli#3967]
### Please confirm that you have:

- [X] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [X] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Theme

### Expected behavior

No error in console after commands

### Actual behavior

~ ❯ shopify version                                                    19:41:24
(node:20345) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Current Shopify CLI version: 3.61.0
~ ❯     

### Verbose output

```shell
~ ❯ shopify version                                                    19:41:24
(node:20345) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Current Shopify CLI version: 3.61.0
~ ❯
```

### Reproduction steps

1. install shopify CLI 3.61.0
2. run any shopify command

### Operating System

Mac OS

### Shopify CLI version (check your project's `package.json` if you're not sure)

3.61.0

### Shell

zsh

### Node version (run `node -v` if you're not sure)

v22.2.0

### What language and version are you using in your application?

ruby 2.6.10p210

**Comment by bigskillet** [comment]
Seeing this as well...

**Comment by mikayilsrt** [comment]
I also have the same problem following an update of the shopify order.

**Comment by tomgreenhill** [comment]
Likewise, seeing this after having upgraded via homebrew to `3.61.0`

**Comment by timdmackey** [comment]
How long does it take for a new version to be available in Homebrew? Version 3.61.1 isn't showing up yet.

**Comment by mikayilsrt** [comment]
> How long does it take for a new version to be available in Homebrew? Version 3.61.1 isn't showing up yet.

I just tried to update and it worked for me.
